src/api/goods.ts

```diff
@@ -1,19 +1,27 @@
 import { Good } from '../types/Good';
-
 // eslint-disable-next-line
 const API_URL = `https://mate-academy.github.io/react_dynamic-list-of-goods/goods.json`;
 
 export function getAll(): Promise<Good[]> {
   return fetch(API_URL)
-    .then(response => response.json());
+    .then(response => {
```

Comment:
Pleasedon't ignore previous comments
![image](https://github.com/mate-academy/react_dynamic-list-of-goods/assets/54936044/57d06d1a-6a66-43cd-a9c2-07d16319f17a)